hs_app_netCDF/models.py

```diff
@@ -75,38 +67,26 @@ def create(cls, **kwargs):
 
     @classmethod
     def update(cls, element_id, **kwargs):
```

Comment:
if you're going to keep one of these methods we can otherwise remove, please add a docstring and explain what it does that the base class implementation doesn't do.